packages/stores/src/query/fns/domain-price.ts

```diff
@@ -0,0 +1,69 @@
+import { ObservableCosmwasmContractChainQuery } from "../cosmwasm/contract-query";
+import { KVStore } from "@keplr-wallet/common";
+import { ChainGetter } from "../../common";
+import { computed } from "mobx";
+import { ObservableChainQueryMap } from "../chain-query";
+import { DomainPrice } from "./types";
+
+export class ObservableQueryDomainPriceInner extends ObservableCosmwasmContractChainQuery<DomainPrice> {
+  constructor(
+    kvStore: KVStore,
+    chainId: string,
+    chainGetter: ChainGetter,
+    protected override readonly contractAddress: string,
+    protected readonly domain: string
+  ) {
+    super(kvStore, chainId, chainGetter, contractAddress, {
+      get_normalized_domain_and_price: { domain: domain },
+    });
+  }
+
+  @computed
+  get isValidDomain(): boolean {
+    if (!this.response || !this.response.data.is_valid_domain) {
+      return false;
+    }
+    return this.response.data.is_valid_domain;
+  }
+
+  get result(): any {
+    if (!this.response || !this.response.data.result) {
+      return {};
+    }
+    return this.response.data.result;
+  }
+  get price(): any {
+    if (!this.response || !this.response.data.result.Success.pricing) {
+      return {};
+    }
+    return this.response.data.result.Success.pricing;
+  }
+}
```

Comment:
Avoid "any" typings. Also data and result can be undefined, so handle that as well.